cmd/dashboard/main.go

```diff
@@ -75,6 +75,10 @@ func main() {
 	resource.RegisterHealthProbes(wsContainer)
 	resource.RegisterReadinessProbes(wsContainer)
 
+	logging.Log.Info("Search extensions")
```

Comment:
I think `logging.Log.Infof("Searching for extensions in the namespace %s", installedNamespace)` would be more useful here